sentry/resource_sentry_metric_alert.go

```diff
@@ -340,6 +344,9 @@ func expandMetricAlertTriggerActions(actionList []interface{}) []*sentry.MetricA
 				action.ID = sentry.String(v)
 			}
 		}
+		if v, _ := actionMap["integration_id"].(int); v != 0 {
```

Comment:
It is a good idea to add a comment so other devs know they should not remove the `, _` part. Or you can explicitly change it to something like `if v, ok := actionMap["integration_id"].(int); ok && v != 0 {`.
Why `, _` is important: https://goplay.tools/snippet/3_uSiMykDlD

Reply:
Woahh thanks so much for the explanation! I think I'll change it to `ok && v != 0`

Reply:
Fixed in https://github.com/Canva/terraform-provider-sentry/pull/39/commits/0fca6862f10e06666d3ba12e8037144ebc18bf3a